### PR TITLE
Tests/avatar

### PR DESCRIPTION
**Pull request by mahboubii**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

### CODE REVIEW

File: src/components/Avatar/__tests__/Avatar.test.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+            "flexBasis": 32,
+            "fontSize": 16,
+            "height": 32,
+            "lineHeight": "32px",
+            "width": 32,
+          }
+        }
+      >
+        <img
+          alt={null}
```
Comment (mahboubii) [inline src/components/Avatar/__tests__/Avatar.test.js:57]: @jaapbakker88 @vishalnarkhede Class component was rendering null value for alt text, should we update this to empty string?

Comment (jaapbakker88) [reply src/components/Avatar/__tests__/Avatar.test.js:57]: yep

**Comment by mahboubii** [comment]
> probably good to also add a test 'should render no initials no image if no image and no name is provided'
already there
https://github.com/GetStream/stream-chat-react/pull/247/files#diff-8d680b512e4e3c0f75db1cbb2fb112afR13

**Comment by vishalnarkhede** [comment]
Shouldn't these PRs be against `2.0/development`?

**Comment by jaapbakker88** [comment]
> Shouldn't these PRs be against `2.0/development`?

No, working against master now.

**Pull request merged**